docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     ports:
       - 8443:8443
     environment:
-      - "db_hostname=dcsa-ec-registry-test-db"
```

Comment:
does the [docker-compose-two-platforms.yml](https://github.com/dcsaorg/DCSA-EBL-Envelope/blob/master/docker-compose-two-platforms.yml) also need to change?

Reply:
I don't think so, it has it's own db

Reply:
docker-compose-two-platforms.yml is now aligned with docker-compose.yml